docs/error-messages/compiler-errors-1/compiler-error-c2323.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Compiler Error C2323"
+description: "Learn more about: Compiler Error C2323"
+ms.date: "03/20/2024"
+f1_keywords: ["C2323"]
+helpviewer_keywords: ["C2323"]
+---
+# Compiler Error C2323
+
+'identifier': non-member operator `new` or `delete` functions may not be declared `static` or in a namespace other than the global namespace.
+
+The `new` and `delete` overload operators must be non-static, defined in the global namespace or as class members.
```

Comment:
To be more precise it should be "operator overloads" and not "overload operators".

In addition, with the placement of the comma, one possible misinterpretation of the sentence is that `new`/`delete` operator overloads which are class members must be non-static. That is false as those can be declared `static` and even if the `static` keyword is not explicitly used, it is always a `static` member.

I will address these issues in a follow-up PR.

Reply:
That's cool. It was just a bit verbose before.

Reply:
Maybe we can split it into 2 lines, one for the case where it is in the global namespace (emphasis must be non-static), and the other is for class members (emphasis optional `static` keyword and it being always `static`).

Reply:
Good idea